### ISSUE TITLE
Where to find the "SQuAD1.1-Zhou" and properly formatted SQuAD2.0 datasets

### ISSUE THREAD

**Issue by redfarg** [BangLiu/ACS-QG#13]
Apologizes if this request is not suitable for an issue.

I'm trying to train/reproduce this model (or rather collection of models), but I'm having a hard time gathering all the necessary datasets. I especially cannot find any dataset fitting "SQuAD1.1-Zhou". I was only able to find [this version](https://datarepository.wolframcloud.com/resources/SQuAD-v1.1) of SQuaAD1.1 - but this doesn't seem to fit the expected input format, as the preprocessor in `FQG_data.py` expects tab-separated lined of at least 10 fields, whereas this dataset only has 4. 
Could anyone give me a pointer to the right dataset? 

**EDIT:** Now also concerning the SQuAD2.0 data set, see my comment below.

**Comment by marcoaleixo** [comment]
Hi @redfarg 
I'm not the maintainer but I found the dataset inside another repo from BangLiu - https://github.com/BangLiu/QG

The link is in the readme. \o

**Comment by redfarg** [comment]
Awesome, thank you so much!

**Comment by redfarg** [comment]
I managed to run the `QG_train_seq2seq `experiment successfully, but I'm now struggling with the training of the entailment model (`experiments_1_ET_train.sh`).

In the paper, it is mentioned that the SQuAD2.0 data set is used for training it. As the [official repo](https://rajpurkar.github.io/SQuAD-explorer/) only offers .json (with multiple questions for a single passage combined in one entry) and the input here is expected to be .tsv, it needs to be reformatted. Thus, the questions:

1) Does anyone know where to find a properly formatted .tsv version of the data set? I tried the version from [Wolfram](https://datarepository.wolframcloud.com/resources/SQuAD-v2.0), but the format is wrong (throws an IndexError immediately on the first line).

2) If not, what is the correct format to use? Specifically, looking at the preprocessing script: 

```
text_a = line[3]
text_b = line[4]
label = line[0]
examples.append(
    InputExample(guid=guid, text_a=text_a, text_b=text_b, label=label))
```
Which of the "text_" fields is the question and which the answer (if it matters). And what is the label? Plus, are any of the other fields relevant at some point later?
